### DISCUSSION TITLE
2 Questions: Are save files influenced this or mods? Also what options are enabled by default?

### DISCUSSION THREAD

**Discussion by gachapengames** [Raytwo/Cobalt#52] [Issues]
Hello. I am wondering if save game files are modified or influenced by this or any mods using it for compatibility reasons? Also, are the QOL options for example enabled by default?

**Comment by Raytwo** [comment]
Oh sorry I did not get a notification for this, Cobalt by itself does not modify your save file in any way but mods such as accessory expansion do.
